src/dtos2/single-language/dataset.ts

```diff
@@ -0,0 +1,14 @@
+import { DatasetInfoDTO, RevisionDTO } from '../dataset-dto';
+
+import { SingleLanguageDimension } from './dimension';
+
+export interface SingleLanguageDataset {
+    id: string;
+    creation_date: string;
+    created_by: string;
+    live?: string;
+    archive?: string;
+    dimensions?: SingleLanguageDimension[];
+    revisions: RevisionDTO[];
+    datasetInfo?: DatasetInfoDTO;
```

Comment:
This is fine for now, but I think the "standard" use case is probably always single language, and the "special" use case would be to return multiple languages?

e.g. I'm nearly always in either en or cy, the only time I might want both at the same time is viewing or exporting the entire nested object?